syntax = "proto3";

import "video.proto";

service Trending {
    rpc GetTrendingVideos (TrendingRequest) returns (TrendingPageVideos);
    rpc GetMultipleTrendingVideos (MultipleTrendingRequest) returns (TrendingPageVideos);
}
message TrendingRequest{
    string trendingId = 1;
}
message TrendingPageVideos{
    repeated VideoData VideoData = 1;
}
message MultipleTrendingRequest{
    repeated TrendingRequest TrendingRequest = 1;
}